client/worker.go

```diff
@@ -33,6 +34,13 @@ import (
 	sapmpb "github.com/signalfx/sapm-proto/gen"
 )
 
+// IngestResponse encapsulates the body of response returned by trace ingest and any error encountered
+// by the worker while reading the body.
+type IngestResponse struct {
+	Body *spanfilter.Map
```

Comment:
What is `spanfilter.Map`? I no longer understand how we will fix this so that we store the proper struct here which is decoded either from JSON or from Protobuf response.

Reply:
@tigrannajaryan as discussed during our call, I've removed the JSON decoding for now. Now, the response bytes and any error encountered while reading the response will be returned.